src/sass/components/_collaborators.scss

```diff
@@ -48,7 +48,10 @@
 // ---------------------------
 .collaborator-card {
   h2 {
+    align-items: center;
+    display: inline-flex;
```

Comment:
@stacyk This seems to break the default truncation, but I think that's an okay trade-off:

<img width="311" alt="Screen Shot 2021-04-26 at 2 32 18 PM" src="https://user-images.githubusercontent.com/552316/116133110-54a2da80-a69c-11eb-9c75-a6c35d701e63.png">

Reply:
interesting … I only tested the modal which broke onto two lines. I forget this one is set up differently with different wrappers. I can at least add some padding on that close button

Reply:
![Screen Shot 2021-04-26 at 2 20 40 PM](https://user-images.githubusercontent.com/1581694/116138752-d3971380-a69a-11eb-9b90-d9fb72f79a90.jpg)

I added a slight box shadow to show it is intentional. Thoughts @jgerigmeyer?

Reply:
@stacyk Looks good! 🚀